ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -712,6 +712,26 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: Ensure Guest Access to Shared Folders Is Disabled
+  platforms: macOS
+  platform: darwin
+  description: Allowing guests to connect to shared folders enables users to access selected shared folders and their contents from different computers on a network
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that disables Guest Account.
+    Graphical Method:
+      Perform the following steps to no longer allow guest user access to shared folders:
+        1. Open System Settings
+        2. Select Users & Groups
+        3. Select the i next to the Guest User
+        4. Set Allow guests to connect to shared folders to disabled
+  query: SELECT 1 WHERE NOT EXISTS( SELECT 1 FROM managed_policies WHERE domain='com.apple.smb.server' AND name='AllowGuestAccess' AND value=1);
```

Comment:
If the profile method works tio disable guest access to shared folders, then this should actually be:
```
SELECT 1 FROM managed_policies WHERE domain='com.apple.smb.server' AND name='AllowGuestAccess' AND value=0;
```
We want to check that the administrator explicitly disabled the setting.